source/Octopus.Client.Tests/Conventions/ClientConventions.cs

```diff
@@ -456,7 +493,7 @@ public void AllAsyncRepositoryInterfacesShouldFollowTheseConventions()
         {
             AsyncRepositoryInterfaceTypes
                 .MustConformTo(Convention.MustLiveInNamespace("Octopus.Client.Repositories.Async"))
-                .AndMustConformTo(Convention.NameMustEndWith("Repository"))
+                .AndMustConformTo(Convention.NameMustEndWith("Repository").Or(Convention.NameMustEndWith("RepositoryBeta")))
```

Comment:
Should it be called `BetaRepository` instead? Then you don't need this extra convention?

Reply:
Good call. I've gone through all the existing `XRepositoryBeta` types and renamed them to `XBetaRepository`, which tidies up this convention test, and means that they'll get covered by all the other convention tests too.